docs/tutorial.rst

```diff
@@ -204,41 +206,82 @@ b) Writing a gain calibration file.
       dd = []
       fl = []
       ch = []
-      for ant in antenna_numbers:
+      for ant in ant_array:
           dd.append(gains[pol][ant])
```

Comment:
Are all these steps necessary for a tutorial? The gains, flags, and chisq's are created in a loop, repackaged in a loop, then transposed in a non-transparent way. Because this is just a demonstration, wouldn't it be easier to just create the arrays as random numbers with the appropriate shapes from the beginning?